### ISSUE TITLE
Handle Javascript files with .mjs extension on Github

### ISSUE THREAD

**Issue by Tim15** [sourcegraph/browser-extensions#9]
Currently, when I open a `.mjs` javascript file on github with the Sourcegraph extension, it says
```
No language server connected
Check langserver.org for language servers
```
However, it's just javascript, so it should go to the javascript language server. Is there a way to alias this, like it does with `.jsx`, etc?

Thanks.

**Comment by sqs** [comment]
Just opened https://github.com/sourcegraph/browser-extensions/pull/10 to fix this on the browser extension side. It may also require a change on the language server side (will see).

**Comment by KattMingMing** [comment]
@Tim15 - This is resolved in v1.7.51

**Comment by Tim15** [comment]
Thanks to everyone for a quick resolution!
